pom.xml

```diff
@@ -40,6 +40,7 @@
     <module>model-providers/vertex-ai</module>
     <module>model-providers/vertex-ai-gemini</module>
     <module>model-providers/watsonx</module>
+
```

Comment:
You need to add `<module>model-providers/jlama</module>` here

Reply:
Ah sorry, I'm stupid, I see it now

Reply:
No worries!